client/components/app.jsx

```diff
@@ -9,17 +9,19 @@ export default class App extends React.Component {
     };
   }
 
-  componentDidMount() {
-    fetch('/api/health-check')
-      .then(res => res.json())
-      .then(data => this.setState({ message: data.message || data.error }))
-      .catch(err => this.setState({ message: err.message }))
-      .finally(() => this.setState({ isLoading: false }));
-  }
+  // componentDidMount() {
+  //   fetch('/api/health-check')
+  //     .then(res => res.json())
+  //     .then(data => this.setState({ message: data.message || data.error }))
+  //     .catch(err => this.setState({ message: err.message }))
+  //     .finally(() => this.setState({ isLoading: false }));
+  // }
 
   render() {
-    return this.state.isLoading
-      ? <h1>Testing connections...</h1>
-      : <h1>{this.state.message.toUpperCase()}</h1>;
```

Comment:
You can add back in the `isLoading` code. After line 23 instead of a semi-colon, us a comma then add your React element `LandingPage` after it. 
![Screen Shot 2020-05-05 at 11 53 35 AM](https://user-images.githubusercontent.com/59104550/81105078-44f91780-8ec8-11ea-9f44-74b7c8d0f7d2.png)